templates/zerver/emails/password_reset.source.html

```diff
@@ -3,34 +3,37 @@
 {% block content %}
 <p>
     {% if no_account_in_realm %}
-        Someone (possibly you) requested a password reset email for {{ email }}
-        on {{ realm_uri }}, but you do not have an
-        active account in {{ realm_uri }}.
+        {% trans %}
+        Someone (possibly you) requested a password reset email for
+        {{ email }} on {{ realm_uri }}, but
+        you do not have an active account in {{ realm_uri }}.
+        {% endtrans %}
 
         {% if accounts %}
             {% if multiple_accounts %}
-            However, you do have active accounts in the following
-            organizations.
-            <ul>
-                {% for account in accounts %}
-                <li>{{ account.realm.uri }}</li>
-                {% endfor %}
-            </ul>
-            You can try logging in or resetting your password in the organization
-            you want.
+                (( _("However, you do have active accounts in the following organizations.") }}
+                <ul>
+                    {% for account in accounts %}
+                    <li>{{ account.realm.uri }}</li>
+                    {% endfor %}
+                </ul>
+                {% trans %}You can try logging in or resetting your password in the organization
+                you want.{% endtrans %}
             {% else %}
-            However, you do have an active account in the {{ accounts[0].realm.uri }}
-            organization; you can try logging in or resetting your password there.
+                {% trans %}However, you do have an active account in the {{ accounts[0].realm.uri }}
+                organization; you can try logging in or resetting your password there.{% endtrans %}
             {% endif %}
         {% endif %}
     {% else %}
-        Psst. Word on the street is that you need a new password, {{ email }}.<br />
-        It's all good. Click here and we'll take care of the rest:<br />
-        <a class="button" href="{{ reset_url }}">Reset password</a>
+        {% trans %}Psst. Word on the street is that you need a new password, {{ email }}.{% endtrans %}<br />
+        {{ _("It's all good. Click here and we'll take care of the rest:") }}<br />
+        <a class="button" href="{{ reset_url }}">{{ _('Reset password') }}</a>
     {% endif %}
 </p>
 <p>
+    {% trans %}
     Thanks,<br />
     Your friends at Zulip HQ
+    {% endtrans %}
```

Comment:
I think it might be better to do each line of the signature block as its own translation, to avoid these strings needing to be duplicated between the text and HTML versions?

Also @rishig I think we should get rid of the "Your friends at Zulip HQ" in this template; we should use the same signatures we use in the other emails, right?

Reply:
We could. I could also make a pass on the emails to make them all less unique / more to the point, if we want to go in that direction.

Reply:
I think just this one is problematic, since "at Zulip HQ" doesn't make sense for non-zulipchat.com.